src/Router.js

```diff
@@ -15,7 +15,7 @@ const Router = () => {
         <Route path="login" element={<Login />} />
         <Route path="signup" element={<SignUp />} />
       </Routes>
-      <Footer />
+      <Footer path="Footer" element={<Footer />} />
```

Comment:
```suggestion
      <Footer />
```
footer는 모든 페이지에서 보여야 하는 컴포넌트기 때문에
경로를 지정해 주지 않아도 됩니다!

Reply:
경로 수정 완료했습니다!